src/go/rpk/pkg/plugin/manifest.go

```diff
@@ -79,7 +80,7 @@ func DownloadManifest(url string) (*Manifest, error) {
 
 	const understand = "2021-07-27"
 	if m.Version != understand {
-		return nil, fmt.Errorf("Plugin manifest indicates version %q, and we can only understand %q! Please update rpk!", m.Version, understand)
+		return nil, fmt.Errorf("plugin manifest indicates version %q, and we can only understand %q! Please update rpk", m.Version, understand)
```

Comment:
I think this has some odd ramifications on the output;
this should be entirely lowercased and a single sentence:
```
plugin manifest indicates version %q, and we can only understand %q; please update rpk"
```